modules/ossm-config-otel.adoc

```diff
@@ -99,41 +99,51 @@ spec:
 Once you verify that you can see traces, lower the `randomSamplingPercentage` value or set it to `default` to reduce the number of requests.
 ====
 
-. Deploy the `bookinfo` application in `bookinfo` namespace:
+. Create the `bookinfo` namespace by running the following command:
 +
 [source, terminal]
 ----
 $ oc create ns bookinfo
 ----
+
+. Depending on the update strategy you're using, enable sidecar injection in the namespace by running the appropriate command(s):
```

Comment:
For one, we should [avoid contractions](https://redhat-documentation.github.io/supplementary-style-guide/#contractions) in our docs, but also I would suggest a wording like this if you mean to tell users to choose whichever substep applies to them:
```suggestion
. Enable sidecar injection in the namespace by performing one of the following steps:
```